website/docs/feature-flag-tutorials/use-cases/security-compliance.md

```diff
@@ -0,0 +1,240 @@
+
+## Configure role-based access control for administrators and developers
+
+Now that you understand how Unleash handles authentication securely, let’s explore what needs to happen once your teams have access to the platform. When enterprise auditors evaluate software tools and development practices, they focus intensely on access control mechanisms.
+
+Consider this scenario: your company is undergoing its annual security audit, and the auditors are specifically examining how feature flags are managed across their software development lifecycle. Without role-based access controls (RBAC), this company would face significant compliance risks and potential security vulnerabilities.
+
+[Role-based access control](/reference/rbac) allows for fine-grained permission management so team members can only access and modify feature flags relevant to their specific roles and responsibilities. During an audit, you'll most likely face questions about your access control systems and security measures. Here’s what auditors will want to see:
+
+-   Only authorized personnel can modify feature flags in production.
+-   All the access control changes are tracked and documented.
+-   An approval chain is in place to change critical features.
+-   Sensitive projects remain visible only to appropriate teams and/or project owners.
+
+Unleash is built with many mechanisms in place to handle all of these scenarios. Our systems enforce strict separation of duties, for example, where developers can create and test flags in development environments, while only senior engineers can promote changes to staging.
+
+-   We leave audit trails, automatically [logging every permission change](#audit-manual-and-automated-events-in-unleash) with detailed user information and timestamps.
+-   You can set up [approval guardrails](#use-a-change-management-workflow-for-auditing) for feature flag updates.
+-   [Project isolation](/reference/project-collaboration-mode) ensures sensitive projects remain hidden from unauthorized users, while teams can only access projects relevant to their work, maintaining clear boundaries between different business units' feature flags.
+
+Let’s look at how Unleash gives you complete control over user roles and permissions. At a high level, there are multiple [predefined roles](/reference/rbac#predefined-roles) in Unleash for you to get started with. Root roles are meant to control permissions to top-level resources, spanning across all projects. Project roles, on the other hand, control permissions for a project, the feature flags, and individual configurations per environment. The diagram below breaks down the root roles, project roles, and their respective permissions. Think of them as very specific responsibilities only certain users have in Unleash that are defined by default.
+
+![The diagram showing the relationship between root roles and project roles in Unleash.](/img/root-and-project-roles.png)
+
+:::note
```

Comment:
Actually now that I see the image and the admonition box together on the live preview, I think it looks a bit strange visually. Maybe it would be better as just regular text? sorry to keep going back and forth on this

Reply:
Agree, the image is not very clear. The thing is that editor can do almost everything that a project role can do. Not sure is something enterprise customers will need. I believe here on best practices and recommendations for big enterprises we should advice to make all users viewers and then have specific permissions on a per-project level, following the least privilege principle.

Reply:
I can remove the diagram and update the "note" to be regular text format. And add in the best practices mentioned (from Gaston).

Reply:
We should keep the diagram, I think. I can make the changes to the text formatting

Reply:
applied in a5f4b4a48cc6b1146934f91d2e1198240aeafdca